mappings/net/minecraft/block/impl/BlockFire.mapping

```diff
@@ -7,10 +7,12 @@ CLASS none/amp net/minecraft/block/impl/BlockFire
 	FIELD f UP Lnone/asv;
 	METHOD a deserializeState (I)Lnone/asj;
 	METHOD a getDropAmount (Ljava/util/Random;)I
+	METHOD a setBurnLevels (Lnone/akw;II)V
```

Comment:
setBurnLevel*s*? Please define the parameters so we can decide on the name further.